### ISSUE TITLE
Translate new Support us Pages

### ISSUE THREAD

**Issue by acrymble** [programminghistorian/jekyll#1943]
The new 'Support us' pages are currently only available in English, as they were undergoing some field testing.

I would now like to translate the individual.md and supporters.md pages into the other three languages. I believe the ipp.md file needs more targeted strategies for each language that reflect the realities in different parts of the world, so I'll wait on that for now.

@jenniferisasi can we arrange a meeting to talk through this and implement as much of it as we can?

**Comment by jenniferisasi** [comment]
@acrymble I have had a packed agenda this week and next week won't be easy either. Do you mind if we start working on the documents as a PR here for now? Do the split and styling at least, make sure it works in all languages (I mean the menu). 

For the IPP targeted strategies we would need to meet/email with each managing editor, I think, or at least the French and Portuguese ones, or maybe global team? I have a grasp of the one for Spanish only

**Comment by acrymble** [comment]
I've made a new branch - support-translate

**Comment by acrymble** [comment]
@jenniferisasi I think we should do Spanish first, since you can do that one yourself. I've added new files to the "support-translate" branch:

/es/donaciones.md
/es/colaboradores.md

These have the English text for the corresponding pages pasted into them, and where I could, I've pasted over the Spanish text (where it was already the same as on the existing page).

I think the easiest thing to do now is to ask you to translate the rest of the text, except for the names of any organisations, which I believe we've agreed to keep in the language of the organisation (e.g., 'Western University Library', 'The British Academy').

Then I think we should speak to @rivaquiroga or someone she designates, to think through how the ipp page can best be presented for our intended audiences in Spanish speaking regions. Does that work?

**Comment by jenniferisasi** [comment]
@acrymble yes that would work for me! Are you doing a PR with the changes? Otherwise I can open it from your changes (learning this now upon checking on the new branch) and go from there.

**Comment by acrymble** [comment]
No I haven't done a pull request yet.

**Comment by acrymble** [comment]
I have translated (I think) the filenames for all of the files we will need. Keeping here for easy reference.

Each language now needs three pages linked to the _includes/menu.html:

* /en/ipp.md
* /en/individual.md
* /en/supporters.md


Spanish

* /es/pia.md
* /es/donaciones.md
* /es/colaboradores.md

French

* /fr/pi.md
* /fr/donations.md
* /fr/nos-soutiens


Portuguese

* /pt/ppi.md
* /pt/doacoes.md
* /pt/apoiadores.md

**Comment by drjwbaker** [comment]
[Corporation for Digital Scholarship](https://digitalscholar.org/) has now joined the IPP!! I'm mindful that these pages are under review/translation, so how best to get a PR in that reflects this?

**Comment by acrymble** [comment]
Thanks @drjwbaker. It's fine to update the various pages and we'll make sure the right ones get added as we translate. It may take us time but it's important to recognise these asap.

**Comment by drjwbaker** [comment]
Made a PR for EN. Added Es/FR/PT to https://github.com/programminghistorian/jekyll/tree/support-translate

**Comment by drjwbaker** [comment]
Just added a new Patron level supporter to the relevant ES/FR/PT pages on the "support-translate" branch.

**Comment by jenniferisasi** [comment]
Proceeding to translate the Spanish files through the the support-translate branch

**Comment by jenniferisasi** [comment]
Oh @acrymble realized we are going to need to edit the Patreon banner/alert in the /donations page with new $ update.

**Comment by acrymble** [comment]
Thank you @jenniferisasi. And yes, I'll make sure to update those when we update Patreon.

I'll speak to @rivaquiroga later this week about the pia.md page for Spanish and then we can finish off this first language.

**Comment by acrymble** [comment]
@jenniferisasi I've spoken to Riva and we're going to first go with the existing page before having some conversations about tailoring IPP to Spanish audiences. So @jenniferisasi can you please help us to translate (or port over already translated bits) of the ipp.md page into a new pia.md page for Spanish? Then we can launch those new pages in Spanish. Let me know if you need any clarification on that.

**Comment by drjwbaker** [comment]
This was the issue I was looking for (and thought was closed) when I opened this PR https://github.com/programminghistorian/jekyll/pull/1982. Oops. If I need to move/park the PR, let me know.

**Comment by acrymble** [comment]
@drjwbaker not needed. We will make sure any new additions get added to newly translated pages/branches. You can ignore this branch.

**Comment by jenniferisasi** [comment]
> @jenniferisasi I've spoken to Riva and we're going to first go with the existing page before having some conversations about tailoring IPP to Spanish audiences. So @jenniferisasi can you please help us to translate (or port over already translated bits) of the ipp.md page into a new pia.md page for Spanish? Then we can launch those new pages in Spanish. Let me know if you need any clarification on that.

I think I know what needs to be done. I am taking note to work on it this week.

**Comment by jenniferisasi** [comment]
@acrymble we now have our 3 pages ready to initiate a PR for someone else in the Spanish team to review.

**Comment by acrymble** [comment]
We now have live /es pages with the new format. Thank you @jenniferisasi and @rivaquiroga.

@walshbr I just noticed the twitter bot tweeting today a link to the old /support page in Spanish. I checked the wiki but I don't see any information on finding the spreadsheet so I can update it. Is this documented somewhere? Or if not, can you help me update the link so we get people going to the right page?

**Comment by spapastamkou** [comment]
The link for the bot is provided in the editors guidelines page on the
website.

**Comment by walshbr** [comment]
Yeah it's view only to the public, but we can edit it because I've added us. I'll add it to the wiki though under technical guidance.

**Comment by jenniferisasi** [comment]
Thanks @acrymble for making that change in the bot upon changing the links.

**Comment by jenniferisasi** [comment]
@acrymble do we want to keep this issue open until we get French and Portuguese translations too?

**Comment by acrymble** [comment]
Let's do them all first.

@spapastamkou I think we should have a conversation next about French and IPP. Do you want to delegate that to someone on your team for us to approach? I know you're quite busy these days, and it doesn't need to be you.

**Comment by spapastamkou** [comment]
I ping @mariechristineb and @grandjeanmartin to help with the translation (thanks!)

I will see who could be available for the discussion FR/IPP and will let you know asap @acrymble

**Comment by mariechristineb** [comment]
Update: I've been working on the translations, *Nos soutiens* is done, the other ones are work in progress. 

FYI @acrymble / @jenniferisasi: on the English IPP page, under *Banded rates for ODA eligible countries*, the CAD amount is missing.

**Comment by acrymble** [comment]
Thanks @mariechristineb

I see you've translated the names of organisations. Just to re-iterate an earlier conversation you may not have seen, we're going to keep organisation names in their local language. This is because it is too difficult to translate into 4 languages for the people (me) who have to maintain these pages.

On the second point, there are no countries that use CAD as their currency that are also ODA eligible, so it didn't seem to make sense to include it. As this is one more thing we have to calculate each time we update everything. But if you think it is important, we can include it. We're trying to make these pages really easy to keep up to date.

**Comment by acrymble** [comment]
@spapastamkou I think we need your input on our IPP frequently asked questions section. It is written with my experience talking to English language universities (mostly Canada, US, UK). It may not be relevant to French speaking universities, so this shouldn't be translated directly:

## Frequently Asked Questions

1. What budgets do other institutions draw this contribution from?
 - Some of our partners have dedicated 'Open Access' budgets to support work such as ours. Others have 'Development' budgets or draw upon training budgets and then use our materials for training workshops. We are also able to invoice your contribution as either a subscription or a donation, as best suits your circumstances.
2. Can you provide usage statistics for my university?
 - No, we don't track users in that way, and we don't have a sign-in system for readers.
3. Will my contribution be used on a publication of my choice?
 - Sponsors contribute to the shared infrastructure used across all of our publications. By becoming an IPP sponsor, you strengthen the resources in your chosen language, as well as those across our offering.

**Comment by spapastamkou** [comment]
@acrymble I believe the translations of the partners' names or at least some of them were already there, so if @mariechristineb has translated some, she surely followed the existing pattern. I do not remember how all this had started, but the idea was to make all this understandable to French speaking users, at least when the terms were already in use in French. Perhaps, as it is understandably difficult to continue doing this for page maintenance reasons, in the future we can leave the names as they are in their native language, as you said. Although  we may still need to translate at least geographical names (city and country) to know where an insitution is. What do you think?

**Comment by mariechristineb** [comment]
@acrymble for the banded rates, I hadn't thought of it like that, I saw it just as a list of conversion rates and thought it was odd that only one spot was missing -  but it totally makes sense that there wouldn't be conversion to or from CAD in this context

As for the translations, I do remember seeing that conversation now, but didn't have it in mind when translating, I was just following the pattern of the already existing page. I would agree with @spapastamkou that names that are in use in French should be translated (such as 'Nouvelle initiative Canadienne en histoire de l'environnement'), and would still translate the name of the country. Otherwise, I do agree with keeping the natives names, in which case the English version should use 'KU Leuven bibliotheken' as well.

**Comment by spapastamkou** [comment]
@acrymble For the FAQ section of the page, parts 2 and 3 can be translated as they are. But...

... but part 1 is the tricky one because I am afraid the rules to use budgets are different, at least in France. First of all, I do not know how things happen in Québec (where I suspect they are as you know them in the rest of Canada); or Belgium, Switzerland, and other French-speaking countries. I will stick to the ones in use in France. 

In France, potentially, what is understood as university funding can come from two sources: 1) university libraries, and 2) research units depending from the university funding themselves than can then allocate ressources further. University libraries in my knowledge have budget for acquiring books or subscribing to journals, but not a mechanism to reinforce open access. So in the case of the PH, this should be something like a donation, since we do not have subscribing fees; but in my knowledge, there is no donation mechanism to these budgets. In the case of research units, money can be spent either for research-related expenses (travel funds, equipment...) or for research-related services (dispense a workshop, a training conference...). In this case, budgetary lines are very strict and also here, in my knowledge, there is no donation mechanism (not to say funding is more and more scarce). There is also funding for publications acquisition, especially if there is a unit library, and in this case rules are the same with the ones in university libraries (of which local research units libraries are part).

Other possible sources of funding in France: Fonds national pour la science ouverte (where we submitted last year but did not have) - we can separately talk about this; perhaps the very large infrastructures [Progedo](http://www.progedo.fr/en/) and [Huma-Num](https://www.huma-num.fr/) but we should write them; and also thought of [médialab](https://medialab.sciencespo.fr/en/) in case they have specific rules - also should write to them. We can discuss this further @acrymble if you want. I can also ask to my Uni's library to further confirm or eventually find out something I do not know about possibilities of funding from libraries. 

For the Support Us page in French, then: Given it could correspond to what is effective in Canada (=> + Québec), that francophone countries may have different rules for research budgets, and France's case, we could translate (or keep existing translation) specifying this is how it works with countries from which are current partners come from, and adding that we are open to discuss with anyone interested in becoming partner following budgetary rules of their country. Can something like this work at least for now?

**Comment by spapastamkou** [comment]
@acrymble I wonder if we cannot ask openly for specific kinds of (in kind) support in the FR page. Copy editing of FR lessons could be one of them once we have the chart ready.

**Comment by drjwbaker** [comment]
- [ ] redirect from https://programminghistorian.org/en/support-us#institutional-partner-programme and https://programminghistorian.org/en/supporters to prevent 404s (closes #2041)

**Comment by acrymble** [comment]
@spapastamkou thanks for that extra context on French universities. We do have 2 IPP supporters from French-speaking universities but none from ones in France, so maybe there are extra challenges, as you have noted. As I understand, (@drjwbaker can correct me), we can 'sell' these as 'subscriptions' just like Elsevier or Taylor Francis sell subscriptions. If that's the case, then maybe we can add a FAQ entry that says "Is this a donation or a subscription fee?" with an answer?

@mariechristineb thank you for your help translating the materials. I have changed organisational names back to their native language but kept ones with French names, as you've suggested. And I'll fix the other language pages too. It looks to me like the remaining bits are:

- line 70-84 on /fr/pi.md
- /fr/donations.md

Is that correct?

**Comment by acrymble** [comment]
Can someone on @programminghistorian/technical-team please show me how to add a re-direct to old pages? I tried looking at https://programminghistorian.org/en/lesson-retirement-policy but I can't seem to find the syntax. Our old support-us pages now have new URLs. So we need to use redirects in case someone finds them.

**Comment by walshbr** [comment]
I think you should be able to put in the metadata at the top of the new page:

redirect_from: /lessons/counting-frequencies-from-zotero-items

Where /lessons/counting-frequencies-from-zotero-items would be replaced by whatever the old URL would be. And you'll need to make sure nothing lives at that old location I think. If that doesn't work let me know what the old and new pages are and I can try to take a look.

**Comment by mariechristineb** [comment]
@acrymble that was correct, indeed - I've now finished the translation on /fr/pi.md and partly translated /fr/donations.md

@programminghistorian/french-team 'donations' sounds really strange to my ears, perhaps a typically québécois hypersensitivity to perceived anglicisms, as I know the term is technically correct in French. I kept it because it was the one we were already using before, but I wouldn't mind switching to 'dons' if you're ok with that

**Comment by mariechristineb** [comment]
@acrymble the translation is pretty much done, but it would be great if someone on the @programminghistorian/french-team could have a look to see if they have any thoughts on the vocabulary and the capitalization of 'programme de Partenariat institutionnel'. Or should this wait for the pull request, since these would be minor changes?

**Comment by acrymble** [comment]
Thank you so much @mariechristineb I really appreciate your help with this. Let's create a pull request to preview everything and we can ask for a review from one of the @programminghistorian/french-team .

I will make a pull request.

**Comment by acrymble** [comment]
We've finished this work now for English, Spanish, French.

I'm going to create a new ticket for Portuguese because it's lower priority, needs slightly different bits, and this thread is now too long.
